apps/frontpage/postcss.config.mjs

```diff
@@ -0,0 +1,8 @@
+/** @type {import('postcss-load-config').Config} */
```

Comment:
`.mjs`?

Reply:
Yes apparently this is the new way of doing it.